ebpf/disk-failure/main.go

```diff
@@ -0,0 +1,116 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+//go:build !cgo
+// +build !cgo
+
+package main
+
+import (
+	"C"
+	"bytes"
+	"encoding/binary"
+	"flag"
+	"github.com/DataDog/chaos-controller/ebpf"
+	"github.com/DataDog/chaos-controller/log"
+	bpf "github.com/aquasecurity/libbpfgo"
+	"github.com/aquasecurity/libbpfgo/helpers"
+	"go.uber.org/zap"
+	"os"
+	"os/signal"
+)
+
+var nFlag = flag.Uint64("p", 0, "Process to disrupt")
+var nPath = flag.String("f", "/", "Filter path")
+
+var logger *zap.SugaredLogger
+
+func main() {
+	// Defined a chanel to handle SIGTERM
+	sig := make(chan os.Signal, 1)
+	signal.Notify(sig, os.Interrupt)
+
+	var err error
+	logger, err = log.NewZapLogger()
+	must(err)
```

Comment:
Why name this `must`?

Reply:
It is completely arbitrary. This function was already defined https://github.com/lizrice/libbpfgo-beginners/blob/main/hello.go#L58.